Main.pas

```diff
@@ -1376,6 +1378,37 @@ procedure TMainForm.WipeBoxes;
 end;
                                    
 
+procedure TMainForm.FirstTime1Click(Sender: TObject);
+const
+    Msg='First Time?'#13 +
+        'Welcome! This is the first time setup:'#13 +
+        ''#13 +
+        '1) Select the Contest you wish to operate.'#13 +
+        '2) Type the exchange you wish to send.'#13 +
+        '3) In the station section replace VE3NEA with your call.'#13 +
+        '4) Select your CW Speed, Tone, and Bandwidth.'#13 +
+        '5) Set Min/Max CW Receive speed using Settings menu.'#13 +
+        '6) Under band conditions you can select any realistic hardships.'#13 +
+        '7) Activity is the average amount of responses you want per CQ.'#13 +
+        '    So if no one responds, you might get twice the number the'#13 +
+        '    following time. This is a pile up trainer after all - have fun.'#13 +
+        '8) Select the time limit you want to run the contest.'#13 +
+        '9) The Run button has a drop down.'#13 +
+        '    - Pile up - Hit Enter or F1 to call CQ to start. Get ready for pileups!'#13 
+        '    - Single Calls - Work one station at a time with no pileups.'#13 +
+        '      contact you.'#13 +
+        '    - Pile up - You need to call CQ to start. hit enter or F1.'#13 +
```

Comment:
The "contact you" seems to be dangling.. And pile up was inserted above. So line 1399 and 1400 need to be deleted right?

I know we discussed the run button. Might already been an issue (I think it is) but we mentioned having to change this run button (to say stop when started, move dropdown to near contest selection, lists contest that are selected above, etc)

Reply:
Good find! All repaired and ready to go. Thank you @K6OK for the wording changes.

Here is the updated output:
![image](https://user-images.githubusercontent.com/106113737/205480733-fa838aa5-d5ec-41c1-940f-251186caab65.png)